src/components/study/display-study-status.tsx

```diff
@@ -45,10 +47,16 @@ const StatusLabels: Partial<Record<AllStatus, StatusLabels>> = {
     'PENDING-REVIEW': { type: 'Proposal', label: 'Under Review' },
 }
 
+const styleStatus = (label: string) => {
+    if ([StatusLabels['RUN-COMPLETE']?.label, StatusLabels['PENDING-REVIEW']?.label].indexOf(label) > -1)
+        return theme.colors.red[10]
+    else return 'dark.8'
+}
+
```

Comment:
a final comment, you don't need to return the actual `them.colors.red[10]`, you can return a string `red.10` and then you wouldn't need to import the theme object.